packages/recoil/recoil_values/Recoil_selector.js

```diff
@@ -782,6 +782,7 @@ function selector<T>(
 
     try {
       result = get({get: getRecoilValue, getCallback});
+      setDepsInStore(store, state, deps, executionId)
```

Comment:
Best if this is done outside of this try/catch block.  We could just call it before the return, which references `depValues`, so it might be a nice logical place for it.